lib/bin/bt_ll_acs_nrf53/include/ble_hci_vsc.h

```diff
@@ -104,6 +104,7 @@ enum ble_hci_vs_tx_power {
 	BLE_HCI_VSC_TX_PWR_Neg16dBm = -16,
 	BLE_HCI_VSC_TX_PWR_Neg20dBm = -20,
 	BLE_HCI_VSC_TX_PWR_Neg40dBm = -40,
+	BLE_HCI_VSC_TX_PWR_INVALID = 126,
```

Comment:
We can't add this here without the controller knowing about it. A user will now expect the controller to recognize 126 as invalid.

Reply:
Good point. Removed.
I did not realize that BLE_HCI_VSC_PRI_EXT_ADV_MAX_TX_PWR_DISABLE is actually used as a parameter/input to the controller